src/webgl/shaders/light.vert

```diff
@@ -59,17 +62,21 @@ void main(void){
     if(i < uPointLightCount){
       vec3 loc = (uViewMatrix * vec4(uPointLightLocation[i], 1.0)).xyz;
       vec3 lightDirection = normalize(loc - mvPosition.xyz);
+      vec3 lightVector = mvPosition.xyz - loc;
 
       float directionalLightWeighting = max(dot(vertexNormal, lightDirection), 0.0);
 
+      float lightDistance = length(lightVector);
+      float fallOffFactor =  1.0 / (constAtt + lightDistance * (linearAtt + quadAtt * lightDistance));
```

Comment:
Is this the correct formula? In your proposal it is written as 
```
  float attenuation = 1.0 / (constantAttenuation + lightDistance * linearAttenuation + (lightDistance * lightDistance) * quadraticAttenuation);
```

Just want to double check.

Reply:
those two formulas are equivalent. i think the 2nd one is more readable, though. i don't think the optimization is necessary here.

Reply:
Yes they are equivalent. I'll rewrite the formula and use the second one though.